lib/xlsx_reader/conversion.ex

```diff
@@ -151,7 +151,11 @@ defmodule XlsxReader.Conversion do
   """
   @spec to_decimal(String.t()) :: {:ok, Decimal.t()} | :error
   def to_decimal(string) do
-    Decimal.parse(string)
+    case Decimal.parse(string) do
+      {:ok, decimal} -> {:ok, decimal}
+      {decimal, _} -> {:ok, decimal}
```

Comment:
To be consistent with our other conversion functions, we should match strictly on `{decimal, ""}` to reject any decimal with trailing characters.

Reply:
Sure, changed! :) I've also reformatted these lines to your library coding standard. :)